_cxx/envoy.mk

```diff
@@ -18,7 +18,7 @@ RSYNC_EXTRAS ?=
   ENVOY_COMPILATION_MODE ?= opt
   # Increment BASE_ENVOY_RELVER on changes to `docker/base-envoy/Dockerfile`, or Envoy recipes.
   # You may reset BASE_ENVOY_RELVER when adjusting ENVOY_COMMIT.
-  BASE_ENVOY_RELVER ?= 0
+  BASE_ENVOY_RELVER ?= 2
```

Comment:
I tried another bazel build flag before but that didn't work hence why it's bump to 2.

This can be reset on the next Envoy version upgrade